polytope/polytope.py

```diff
@@ -570,6 +564,18 @@ def _rotate(polyreg, i=None, j=None, u=None, v=None, theta=None, R=None):
     return R
 
 
+def _Givens_rotation_matrix(i, j, theta, N):
```

Comment:
If all these rotation functions are added to a public API, shouldn't they be in their own module instead of with the `Polytope` class definitions?

Givens is the name of the person who introduced this rotation matrix, and as far as I know, people refer to it as the Givens rotation matrix. That's why it is named the way it is.